client/tox.ini

```diff
@@ -34,6 +38,12 @@ depends = py37
 parallel_show_output = true
 
 [testenv:check]
+deps =
+    black
+    flake8
+    flake8-black
+    isort
+    pip-compile-multi
```

Comment:
Do you still need this call to pip-compile-multi if there are no longer any requirements files ??

Reply:
Good catch, we certainly don't.